.golangci.yml

```diff
@@ -56,7 +56,7 @@ run:
 linters-settings:
   dupl:
     # tokens count to trigger issue, 150 by default
-    threshold: 25
+    threshold: 150
```

Comment:
@stepanvalyavskiy why so big change here?

Reply:
It was wrongly configured as 15, but default is 150.
25 triggers even 1 live delegating in logger functions.
Let set to 30, but I afraid we will need to increase it later once again.

Reply:
@stepanvalyavskiy I see, then it's ok to have 150 as default value